gateway-ui/.storybook/main.ts

```diff
@@ -25,6 +25,21 @@ const config: StorybookConfig = {
       },
     },
   },
+  webpackFinal: async (config) => {
+    return {
+      ...config,
+      resolve: {
+        ...config.resolve,
+        fallback: {
+          "os": require.resolve("os-browserify/browser"),
+          "crypto": require.resolve("crypto-browserify"),
+          "path": require.resolve("path-browserify"),
+          "vm": require.resolve("vm-browserify"),
+          "stream": require.resolve("stream-browserify")
```

Comment:
The `DidRegistry` ts-client has some nodejs dependencies that are not present in the browser. This webpack config allows those packages to correctly resolve. We should consider making the `DidRegistry` package support both the browser and nodejs.

Reply:
Now that we will be adding a webpack bundler to the UI component this issue goes away.